common/src/main/java/org/conscrypt/ConscryptSocketFactory.java

```diff
@@ -19,21 +19,22 @@
 import java.io.IOException;
 import java.net.InetAddress;
 import java.net.Socket;
+import java.net.SocketException;
 import java.net.UnknownHostException;
 import java.security.KeyManagementException;
 import javax.net.ssl.SSLSocketFactory;
 
 /**
  * An implementation of {@link SSLSocketFactory} based on BoringSSL.
  */
-final class OpenSSLSocketFactoryImpl extends SSLSocketFactory {
+final class ConscryptSocketFactory extends SSLSocketFactory {
```

Comment:
This can't really be renamed easily because of GmsCore's `ProviderInstallerImpl`. It needs to have this and `OpenSSLServerSocketFactoryImpl` remain the same name for reflective calls.